doc/dev/generate.md

```diff
@@ -0,0 +1,94 @@
+        "@azure-tools/typespec-azure-core":
+            azure-resource-provider: "Microsoft.<ProviderName>"
+    ```
+
+-   **Naming** – Follow the `azure_mgmt_<service>` crate pattern and mirror the service folder layout used by other languages (`sdk/<service>/<crate-name>`). When in doubt, look at the Go SDK’s management plane docs for a matching service and port the structure.
+-   **Spec changes** – Capture the `azure-rest-api-specs` commit SHA (or PR branch commit) that contains the TypeSpec updates; you will reference it from `tsp-location.yaml`.
+
+## Repository setup checklist
+
+The base client guide covers initialization in detail. For management plane, double-check these repo-specific tasks once your service directory exists:
+
+-   Add `sdk/<service>/<crate-name>` to the workspace members in the root `Cargo.toml`, but keep it out of `default-members`.
+-   Drop a `ci.yml` into `sdk/<service>/` if the service folder is new so the CI pipeline can pick the crate up. Copy an existing management plane `ci.yml` as the starting point.
+-   Create `tsp-location.yaml` beside the crate with `repo`, `commit`, and `directory` keys pointing at the Resource Manager TypeSpec project.
+
+## Running `tsp-client`
+
+Use the same commands outlined in the base guide—`tsp-client init` for brand-new crates, `tsp-client update` for refreshes. The only management-plane twist is path selection:
+
+```powershell
+# Example update call
+tsp-client update https://github.com/Azure/azure-rest-api-specs/blob/<commit>/specification/<service>/<project>/tspconfig.yaml
+
+# Local iteration
+$spec = "<your-repo-path>/Azure/azure-rest-api-specs/specification/<service>/<project>"
+tsp-client update "$spec/tspconfig.yaml"
+
+```
+
+Generation writes into `src/generated/`; keep the directory read-only and place any ergonomic layers or custom entry points next to `lib.rs`, exactly as described in the shared guide.
+
+## Management-plane finishing work
+
+Most polish items match the data-plane flow, but a few deserve special emphasis for Resource Manager crates:
+
+-   **Cargo metadata** – Ensure `Cargo.toml` advertises the crate as “Azure Resource Manager” in `description` and includes `[lints] workspace = true`.
+-   **Feature gating** – Resource Manager specs often bundle GA and preview APIs. Expose preview-only operations behind an opt-in feature (`cfg(feature = "preview")`). Document the feature in the crate README.
+-   **Credential surface** – Make sure the public constructors accept `azure_identity` credentials and expose subscription-scoped builders, matching existing management plane crates.
+-   **Integration tests** – Provision test resources with `eng/common/TestResources/New-TestResources.ps1 -ServiceDirectory <service>` and record with `#[recorded::test]`. Management plane tests frequently require role assignments, so verify the Bicep template grants the test application proper `Microsoft.Authorization/roleAssignments` permissions.
```

Comment:
This topic may need it's own doc. 
Do teams even create custom tests for their mgmt plane libraries? I not familiar with this in the mgmt plane space but I feel like this is more involved than what most service teams go through to release mgmt plane libraries.